packages/shopify-codemod/test/fixtures/mocha-context-to-global-references/rename.output.js

```diff
@@ -0,0 +1,17 @@
+suite('test', function() {
+  afterEach(function() {
+    sinon.stub(foo, 'bar');
+  });
+
+  before(function() {
+    this.testSpy = sinon.spy();
```

Comment:
From what I've seen of the tests, `this.spy` is going to call into a sandbox.  Is there a risk of this transform changing teardown behaviour?